src/routes/clubs.$slug.manage.tsx

```
@@ -38,7 +38,9 @@ import { ClubAnalyticsDashboard } from "@/components/clubs/ClubAnalyticsDashboar
 import { PermissionsGrid } from "@/components/Clubs/PermissionsGrid";
 import ClubRenewalWizard from "@/components/ClubRenewalWizard";
 import { ClubFinancesTab } from "@/components/Clubs/ClubFinancesTab";
-import { HandoverChecklist } from "@/components/club/HandoverChecklist";
+import { SponsorshipTiersManager } from "@/components/sponsorship/SponsorshipTiersManager";
+import DollarSign from "lucide-react/dist/esm/icons/dollar-sign";
+import Handshake from "lucide-react/dist/esm/icons/handshake";import { HandoverChecklist } from "@/components/club/HandoverChecklist";
```

ESLint

[error] Insert `⏎`

(prettier/prettier)